vispy/plot/plotwidget.py

```diff
@@ -338,17 +341,99 @@ def volume(self, vol, clim=None, method='mip', threshold=None,
 
     def surface(self, zdata, **kwargs):
         """Show a 3D surface plot.
-        
+
         Extra keyword arguments are passed to `SurfacePlot()`.
-        
+
         Parameters
         ----------
         zdata : array-like
             A 2D array of the surface Z values.
-            
+
         """
         self._configure_3d()
         surf = scene.SurfacePlot(z=zdata, **kwargs)
         self.view.add(surf)
         self.view.camera.set_range()
         return surf
+
+    def colorbar(self, cmap, orientation="",
+                 label="", clim=("", ""),
+                 border_width=0.0, border_color="black",
+                 **kwargs):
+        """Show a ColorBar
+
+        Parameters
+        ----------
+        cmap : str | vispy.color.ColorMap
+            Either the name of the ColorMap to be used from the standard
+            set of names (refer to `vispy.color.get_colormap`),
+            or a custom ColorMap object.
+            The ColorMap is used to apply a gradient on the colorbar.
+        orientation : {'left', 'right', 'top', 'bottom'}
```

Comment:
I think the `Widget` instead of having `orientation` as the user-facing parameter, should probably have `location` instead. I think they are pretty much the same thing, but users in `plot` probably want to think about where their colorbar will be placed, whereas at the `scene` level they are probably both orienting it and placing it somewhere themselves.